### ISSUE TITLE
ANSI control codes included in the identity

### ISSUE THREAD

**Issue by ilesinge** [RedpointGames/pkgsign#3]
When signing with Keybase, the output of the "keybase id" includes ansi codes, not only my keybase username. These ansi codes are captured and stored in the signature.json file:
```
"identity": {
    "keybaseUser": "\u001b[1milesinge\u001b[22m"
}
```
This provokes an error when verifying the package because the ansi codes are used when constructing the keys URL:
```
fetching public keys of user ilesinge...
https://keybase.io/ilesinge/pgp_keys.asc
TypeError: Request path contains unescaped characters
    at new ClientRequest (_http_client.js:52:11)
    at Object.exports.request (http.js:31:10)
    at exports.request (https.js:206:15)
    at /home/nous/Development/pkgsign/node_modules/node-fetch/index.js:119:13
    at new Fetch (/home/nous/Development/pkgsign/node_modules/node-fetch/index.js:49:9)
    at Function.Fetch [as default] (/home/nous/Development/pkgsign/node_modules/node-fetch/index.js:37:10)
    at KeybaseVerifier.<anonymous> (/home/nous/Development/pkgsign/dist/lib/keybaseVerifier.js:24:57)
    at next (native)
    at /home/nous/Development/pkgsign/dist/lib/keybaseVerifier.js:7:71
    at __awaiter (/home/nous/Development/pkgsign/dist/lib/keybaseVerifier.js:3:12)
error Command failed with exit code 1.
```
(not visible in this snippet but "ilesinge" is in bold in the URL in my terminal)

**Comment by ilesinge** [comment]
I guess ansi codes should be removed here https://github.com/RedpointGames/pkgsign/blob/master/src/lib/keybaseSIgner.ts#L21 using something like https://github.com/chalk/strip-ansi.
Sadly, keybase doesn't seem to provide a clean way to get only the username (for now).
By the way my OS is Linux.

**Comment by hach-que** [comment]
Thanks for this report, I will look at this soon.

**Comment by ilesinge** [comment]
If you validate a solution between the strip-ansi package OR manual stripping with a fat regexp, I can bake a pull request for this.

**Comment by hach-que** [comment]
@ilesinge I've pushed a fix out on `master` which uses `strip-ansi` to resolve the issue. If you could verify that the issue is resolved on master, I'll go ahead and release 0.1.3 once I get back from dinner.

**Comment by ilesinge** [comment]
It works!

"identity": {
    "keybaseUser": "ilesinge"
  }
